### ISSUE TITLE
Paper Mario Sticker Star gets stuck in a black screen

### ISSUE THREAD

**Issue by Adamillo** [Panda3DS-emu/Panda3DS-Games-List#86]
### Game Name (and version if necessary)

Paper Mario Sticker Star

### Hardware Specs (If necessary)

CPU: Intel Core i5-7360U
GPU: Intel Iris Plus Graphics 640
RAM: 8GB

### Operating System

Windows 10

### Game State

Title explains everything

**Comment by wheremyfoodat** [comment]
This is a feature

**Comment by Adamillo** [comment]
TRUE! On one hand, the game is unplayable in Panda3DS. On the other hand, this is Paper Mario Sticker Star! This game fucking sucks LMAO

**Comment by Enderspearl184** [comment]
can you even call it a game though its more of a method of torture tbh
